docs/guides/1228-go-ci.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: /1228/go-ci
+displayed_sidebar: europa
+---
+
+# Go project CI
```

Comment:
Title is unclear. What is this guide about? What will I learn? Something like "Testing a Go project"

Reply:
I am also wondering how this relates to https://github.com/dagger/dagger/blob/main/docs/use-cases/1219-go-docker-hub.md

Reply:
done

Reply:
Sorry, didn't see your comment Gerhard.
Somehow, this documentation is not indexed on the docs website. It's not on the sidebar, and I tried to look for "go docker hub" in the search and it didn't find this article. Is it in some draft?
I could access it via the slug, though.

Reply:
Yes, it is a draft post. It needs more work before we can publish it. It would make sense to capture it in a new, hidden category, something like https://docs.dagger.io/drafts. @grouville has it almost figured out in #2152.